src/Symfony/Bridge/PhpUnit/bootstrap.php

```diff
@@ -12,11 +12,9 @@
 use Doctrine\Common\Annotations\AnnotationRegistry;
 use Symfony\Bridge\PhpUnit\DeprecationErrorHandler;
 
-// Detect if we're loaded by an actual run of phpunit
-if (!defined('PHPUNIT_COMPOSER_INSTALL') && !class_exists('PHPUnit_TextUI_Command', false) && !class_exists('PHPUnit\TextUI\Command', false)) {
```

Comment:
removing this is broken. The logic will now register the bridge error handler even when the bootstrap.php file was not loaded during a composer run but during a normal usage of the app using the composer autoloader.